modules/ibi-install-lcao-cli.adoc

```diff
@@ -0,0 +1,111 @@
+// Module included in the following assemblies:
+// * edge_computing/ibi-image-based-install.adoc 
+
+:_mod-docs-content-type: PROCEDURE
+[id="ibi-install-lcao-cli_{context}"]
+= Installing the {lcao} by using the CLI
+
+You can use the OpenShift CLI (`oc`) to install the {lcao} from the 4.15 Operator catalog.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Create a namespace for the {lcao}.
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openshift-lifecycle-agent
+  annotations:
+    workload.openshift.io/allowed: management
+----
+
+.. Create the `Namespace` CR:
++
+[source,terminal]
+----
+$ oc create -f lcao-namespace.yaml
+----
+
+. Create an Operator group for the {lcao}.
++
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: openshift-lifecycle-agent
+  namespace: openshift-lifecycle-agent
+spec:
+  targetNamespaces:
+  - openshift-lifecycle-agent
+----
+
+.. Create the `OperatorGroup` CR:
++
+[source,terminal]
+----
+$ oc create -f lcao-operatorgroup.yaml
+----
+
+. Create a `Subscription` CR:
+
+.. Define the `Subscription` CR and save the YAML file, for example, `lcao-subscription.yaml`:
++
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1
+kind: Subscription
+metadata:
+  name: openshift-lifecycle-agent-subscription
+  namespace: openshift-lifecycle-agent
+spec:
+  channel: "alpha"
```

Comment:
This should be `channel: "stable"` from 4.16

[From IBU Docs](https://github.com/openshift/openshift-docs/pull/74025/files#diff-b1a209cc7b9e9c35b6310de9c963700c10d7b7c68c0a0aad3a5471fc722f80e9R69)